build-tools/rel-build.sh

```diff
@@ -23,7 +23,7 @@ export GOPATH=/build
 if [ "$COVERALLS_TOKEN" ]; then
   cat $BUILDDIR/coverage/merged-coverage.out >> $BUILDDIR/merged-coverage.out
   cat $BUILDDIR/python-coverage.txt >> $BUILDDIR/merged-coverage.out
-  goveralls \
-    -coverprofile=$BUILDDIR/merged-coverage.out \
-    -service=travis-ci
+  # goveralls \
+  #   -coverprofile=$BUILDDIR/merged-coverage.out \
+  #   -service=travis-ci
```

Comment:
This shouldn't go in

Reply:
Your right, it wasn't supposed to be part of the review